mods/fire/init.lua

```diff
@@ -3,7 +3,12 @@
 -- Global namespace for functions
 
 fire = {}
-
+fire.features = {
```

Comment:
most mod just support version of minetest (and minetest game) which have the features they need and crash when it doesn't.

iirc, there no way to override an abm and trying to support older version without this feature by adding an abm would just make it an abm race to see which get the node first

the only way is for them to override the abm in the code on their server when using an older version,

the only thing useful about this which i can think of is it let mod print a pretty message before exiting instead of crashing